docs/docs-ref-autogen/preview-apis.md

```diff
@@ -12,19 +12,16 @@ New Office Scripts APIs are first introduced in "preview", and then later releas
 > [!IMPORTANT]
 > Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use preview APIs in a production environment or within business-critical documents.
 
-The following table provides a concise summary of the APIs, while the subsequent [API list](#api-list) section gives a detailed list.
-
-| Feature area | Description | Relevant objects |
-|:--- |:--- |:--- |
-| Save and email a PDF | Save a worksheet as a PDF and email it to yourself or your team. For a sample, see [Save a worksheet and email it as a PDF](/office/dev/scripts/resources/samples/save-as-pdf-email-as-pdf). | [EmailAttachment](/javascript/api/office-scripts/officescript/officescript.emailattachment), [MailProperties](/javascript/api/office-scripts/officescript/officescript.mailproperties) |
-
 ## API list
 
 Office Scripts APIs use two namespaces: `OfficeScript` for APIs that are not connected to an Excel workbook, and `ExcelScript` for APIs that work with Excel workbooks. Currently, the only preview APIs are in the `OfficeScript` namespace. The following section lists those APIs.
```

Comment:
Do we need this explanation anymore? I think we could just have the API List header, followed by the note.